test/controllers/user.test.js

```diff
@@ -0,0 +1,68 @@
+const supertest = require('supertest');
+const models = require('../../app/models/index');
+const app = require('../../app');
+
+const request = supertest(app);
+const userAttributes = {
+  firstName: 'Omar',
+  lastName: 'Rodriguez',
+  email: 'omar.rodriguez@wolox.com',
+  password: 'password1923'
+};
+
+describe('usersController.signUp', () => {
+  it('Creates a user', () =>
+    request
+      .post('/users')
+      .send(userAttributes)
+      .then(response => {
+        expect(response.text).toBe('Omar');
```

Comment:
Check the signup PR. After fixing it, this should be calling the first_name property